editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid

```diff
@@ -20,8 +20,8 @@ The ''action-createtiddler'' widget is invisible.
 
 |!Attribute |!Description |
 |$basetitle |The initial title that will be attempted. If a tiddler with that title already exists, then a numerical counter is added to the title and incremented until it is unique|
-|$savetitle |<<.deprecated-since "5.1.20" "ActionCreateTiddlerWidget Example 5">> A text reference identifying a field or index into which the title of the newly created tiddler will be stored after it is created |
```

Comment:
Why the change in the deprecation cutoff version?

Reply:
See: https://tiddlywiki.com/archive/full/TiddlyWiki-5.1.20.html#ActionCreateTiddlerWidget `$savedrafttitle` was new in 5.1.20
See: https://tiddlywiki.com/archive/full/TiddlyWiki-5.1.23.html#ActionCreateTiddlerWidget where both settings where deprecated. 

It probably was a "copy paste" error using the version number from `$savedrafttitle` as deprecation version, which is wrong.

Reply:
updated the link to v5.2.0  https://tiddlywiki.com/archive/full/TiddlyWiki-5.2.0.html#ActionCreateTiddlerWidget